api/config/config.js

```diff
@@ -0,0 +1,18 @@
+const { MYSQL } = require('./credentials');
+
+const {
+    DB_HOST,
+    DB_USERNAME,
+    DB_PASSWORD,
+    DB_NAME
+} = MYSQL
+
+module.exports = {
+    'development': {
+        'username': `${DB_USERNAME}`,
+        'password': `${DB_PASSWORD}`,
```

Comment:
I suggest enforcing uniform properties here for this object so that it's not inconsistent some properties having quotes and some don't. The only time quotes are ever really needed is if the property key has a space or hyphen or other special character